policy/common/methods.go

```diff
@@ -72,6 +73,29 @@ func (m *Methods) Candidates(ctx context.Context, prctx pull.Context) ([]*Candid
 		}
 	}
 
+	if len(m.BodyPatterns) > 0 {
+		prBody, err := prctx.Body()
+		if err != nil {
+			return nil, err
+		}
+		if m.BodyMatches(prBody.Body) {
+			createdAt := prBody.CreatedAt
+			lastEditedAt := prBody.LastEditedAt
+			var updatedAt time.Time
+
+			if !lastEditedAt.IsZero() {
+				updatedAt = lastEditedAt
+			} else {
+				updatedAt = createdAt
+			}
```

Comment:
style: I think this could be simplified:

```suggestion
			updatedAt := prBody.LastEditedAt
			if updatedAt.IsZero() {
				updatedAt = createdAt
			}
```